test/e2e/node/container_probe.go

```diff
@@ -14,17 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package node
```

Comment:
Will change the directory name.
Yeah, and at least till now, shared test is still minority. Maybe we could do that in the future. :)